packages/api/src/controllers/region.js

```diff
@@ -0,0 +1,57 @@
+import Router from 'express/lib/router'
+import { authMiddleware } from '../middleware'
+
+const app = Router()
+
+const getOrchestrators = async (req, res, next) => {
+  const orchestrators = await req.getOrchestrators(req)
+
+  return res.json(orchestrators.map(({ address }) => ({ address })))
+}
+
+app.get('/:region', authMiddleware({ admin: true }), async (req, res, next) => {
+  const region = await req.store.get(`region/${req.params.region}`)
+  return res.json(region)
+})
+
+app.post(
```

Comment:
We should make a JSON-Schema validator for this and do a `validatePost()` middleware.

Reply:
I also think this should be a PUT that creates the region if it doesn't exist, rather than a POST that makes a new region every time. The use case here is each region continually pushing its O list to the server here, we won't need to create a lot of new regions with new ids.

Reply:
(But `validatePost()` should still work to validate even though it's a PUT)